html/api.html

```diff
@@ -1204,12 +1204,20 @@ <h2 class="sub-header">
 <div class="alert alert-success" role="alert">This API is open and can be accessed without any restrictions!</div>
 <p>Usage-Description</p>
 <ul class="request">
-	<li><div class="tabcol0">key0</div>
-		<div class="tabcol1">&nbsp;=&nbsp;&lt;value0&gt;</div>// description0</li>
-	<li><div class="tabcol0">key1</div>
-		<div class="tabcol1">&nbsp;=&nbsp;&lt;value1&gt;</div>// description1</li>
-	<li><div class="tabcol0">key2</div>
-		<div class="tabcol1">&nbsp;=&nbsp;&lt;value2&gt;</div>// description2</li>
+	<li><div class="tabcol0">text</div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;text&gt;</div>//Value of text like Hello </li>
+	<li><div class="tabcol0">mlat</div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;mlat&gt;</div>// Latitude Value</li>
+	<li><div class="tabcol0">mlon</div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;mlon&gt;</div>// Longitude Value</li>
+	<li><div class="tabcol0">zoom</div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;zoom&gt;</div>// Zoom Value</li>
+	<li><div class="tabcol0">width</div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;width&gt;</div>// Width Value</li>
+	<li><div class="tabcol0">height</div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;height&gt;</div>// Height Value</li>
+	<li><div class="tabcol0"></div>
+		<div class="tabcol1">&nbsp;=&nbsp;&lt;&bbox='<maplongWest>','<maplatSouth>','<maplongEast>','<maplatNorth>';&gt;</div>// BBOX parameter</li>
```

Comment:
put the `bbox` in `tabcol0` class and the rest without `>` or `<` in `tabcol1` which makes it looks like
`maplongWest,maplatSouth,maplongEast,maplatNorth`